Remotion/Web/Development.WebTesting.IntegrationTests/IntegrationTest.cs

```diff
@@ -44,6 +45,11 @@ protected WebTestHelper Helper
       get { return _webTestHelper; }
     }
 
+    protected IDriver Driver

```

Comment:
[canbenull]

Reply:
There are no JetBrains annotations in this specific assembly. Should a dependency be added or continued without the attribute?